### PR TITLE
UHF-9032: TPR Unit additional info in accordions

### PR DESCRIPTION
**Pull request by dire**
# [UHF-9032](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9032)

## What was done

* New TPR data is show as accordions in TPR units pages.

## How to install

* Make sure your instance (some instance with TPR units) is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-9032_tpr-unit-accordions`
* Run `make drush-cr drush-locale-update`

## How to test

* [x] Check some unit pages and that the new TPR fields (Topical, Charges, Further information, Other contact information and Web sites) are shown nicely inside accordions. More info about how the data should be divided into accordions is in the ticket.
* [x] Check that there is no strange gaps between the accordion heading and the content (should be continuous grey background.
* [x] Check that the translations work in the accordion headers.
* [x] Check that the code follows our standards

## Designers review

* [x] This PR does not need designers review
* [ ] This PR has been visually reviewed by a designer (Name of the designer)

[UHF-9032]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9032?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/scss/06_components/paragraphs/_accordion.scss [hunk 1]
```diff
@@ -137,6 +137,10 @@
   > *:first-child {
     margin-top: 0;
   }
+
+  > .component--paragraph-text .component__container .component__content > .form-item:first-child > *:first-child {
```
Comment (teroelonen) [inline src/scss/06_components/paragraphs/_accordion.scss:141]: Oh boy this is quite hard to read 😂 Are you sure this is the only way to target what ever you are targeting? At least we need comment here that what does this do.

**Review by teroelonen — CHANGES_REQUESTED** [review]
Just a small clarification needed on the css!

File: src/scss/06_components/paragraphs/_accordion.scss (hunk 1 above)
Comment (dire) [reply src/scss/06_components/paragraphs/_accordion.scss:141]: Yeah it's quite a monster, isn't it..
I'll try to find better selector and if can't then I'll add a comment.
There was an unnecessary gap between the accordion heading and the content caused by the `<p>` tag's margin, but it should be only removed from the first one.

Comment (dire) [reply src/scss/06_components/paragraphs/_accordion.scss]: @teroelonen I made the selector little less complicated and added a comment. If a better solution pops up in your mind, please tell! 🙂

Comment (teroelonen) [reply src/scss/06_components/paragraphs/_accordion.scss]: @dire this looks like something I can still manage - not easy but definitely better than before and the comment makes it easier to understand. Good work 👍

**Review by teroelonen — APPROVED** [review]
This looks good now! Nice work 🦖

**Pull request merged**